src/Http/Controllers/Corporation/WalletController.php

```diff
@@ -24,7 +24,7 @@
 
```

Comment:
# Corporation Wallet

## Transaction
- sorting by total is not working properly (is there a missing attribute ?)